### ISSUE TITLE
realloc fail?

### ISSUE THREAD

**Issue by bw1** [brentjohnson/xconq#5]
kernel/supply.c row 382
   
```c
if (!realloc(mstats, (size_t)ms - (size_t)mstats)) {
	Dprintf("realloc() failed; exiting");
	exit(1);
}
```
[7193d248c80b727a88789724116ae102cfd8600a]

this thing break the startup

**Comment by brentjohnson** [comment]
Should be fixed with PR#6

**Comment by bw1** [comment]
thx
